lib/tasks/testall.js

```diff
@@ -21,7 +22,10 @@ module.exports = CoreObject.extend({
           //create a fake promise for consistency
           promise = RSVP.Promise.resolve();
         } else {
-          promise = BowerHelpers.cleanup(task.project.root);
+          promise = RSVP.all([
```

Comment:
I can't comment on the exact line because github won't let me, but above there's a `backupBowerFile` call that might need to be accompanied by a `backupNpmFile`?